new_platforms/vsextension/ProjectWizard/ImportEnclaveCommand.cs

```diff
@@ -310,8 +311,16 @@ private async void Execute(object sender, EventArgs e)
                             config3.SetPropertyValue("Configuration", true, "WindowsSDKDesktopARMSupport", "true");
                             config3.SetPropertyValue("Configuration", true, "WindowsSDKDesktopARM64Support", "true");
                         }
+
+                        // Change OutDir to $(SolutionDir)bin\$(Platform)\$(Configuration)\
+                        // so it's the same as the enclave.
+                        config.OutputDirectory = "$(SolutionDir)bin\\$(Platform)\\$(Configuration)\\";
```

Comment:
If the host project that the enclave is imported into is built during the developer's CI infrastructure, changing the output directory runs the risk of breaking any such automation. Also, if the developer changes the enclave's output directory before importing it into a host, this hardcoded string won't match.

Would it make sense to give the user the option to choose between:

1. Changing the host's output directory to match the enclave's (not hardcoded);
2. Changing the enclave's output directory to match the host's (idem);
3. Doing nothing.

What does Intel's extension do in this regard?

Reply:
I think Intel's does nothing so it won't run until you do stuff manually.  Yes it would make sense to be more intelligent in the future, but I leave that for a future change.